app/file_sender.py

```diff
@@ -77,11 +114,18 @@ def check_partial_files(partial_files_dir: Path):
         action_type, pack_code, batch_id, batch_quantity = get_metadata_from_partial_file_name(print_file.name)
         actual_number_of_lines = sum(1 for _ in print_file.open())
         if int(batch_quantity) == actual_number_of_lines:
+            context_logger = logger.bind(action_type=action_type.value,
+                                         pack_code=pack_code.value,
+                                         batch_id=batch_id,
+                                         batch_quantity=batch_quantity)
+
+            context_logger.info('Checking complete file for duplicates')
             if not check_partial_has_no_duplicates(print_file, pack_code):
-                logger.warn('Quarantining print file with duplicates', partial_file_name=print_file.name)
+                context_logger.warn('Quarantining print file with duplicates')
```

Comment:
`print_file.name` removed from log?

Reply:
`context_logger` already has all the data in the file name set to log on every line